libflux/src/flux/benches/everything.flux

```diff
@@ -0,0 +1,5000 @@
+@tableflux.h2o_temperature{ state, time > -1y }
```

Comment:
the syntax here is wrong. In fact it looks like its the proposed Table Flux syntax. The parser will error on this syntax. So long as that doesn't short circuit parsing the rest of the file I am ok with it. Otherwise we should make sure the file parses without error.

Reply:
Yeah, I saw that and was like "Oh, I didn't know that worked." I think it'd be better to just use valid flux, so I've narrowed down how the super file gets created and am running the test now.

Reply:
You can ignore the entire `/colm` directory when building the input.

Reply:
I just narrowed it down to everything in `stdlib`